### PR TITLE
UX Improvements

### PR DESCRIPTION
**Pull request by lciolecki**
screenshot-localhost_3000-2021 02 17-12_43_09

### CODE REVIEW

File: ui/client/containers/Processes.tsx [hunk 1]
```diff
@@ -49,14 +56,16 @@ const ElementsRenderer: RowsRenderer = ({processes, getProcesses, statuses}) =>
...
-const sortable = ["name", "category", "modifyDate", "createdAt", "createdBy"]
+const sortable = ["name", "category", "modifyDate", "createdAt", "createdBy", "actionDate", "actionUser"]
 const filterable: Filterable = ["name", "processCategory", "createdBy"]
 const columns = [
   {key: "name", label: "Name"},
   {key: "category", label: "Category"},
   {key: "createdBy", label: "Created by"},
   {key: "createdAt", label: "Created at"},
   {key: "modifyDate", label: "Last modification"},
+  {key: "actionDate", label: "Last action date"},
```
Comment (mproch) [inline ui/client/containers/Processes.tsx:67]: Is it understandable? I mean, one does not see what the action actually was? I know that it's somehow reflected in state, but I wonder if we can make it more clear? Wouldn't some tooltips be enough?

Comment (lciolecki) [reply ui/client/containers/Processes.tsx:67]: Hmm, I can add some tooltip with information: Process was deploy/canceled/archived/unarchived at  / by?

Comment (mproch) [reply ui/client/containers/Processes.tsx:67]: I was more thinking about tooltip with action data by status icon

File: ui/client/components/Process/ProcessLastAction.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+export default function ProcessLastAction(props: Props): JSX.Element {
+  const {t} = useTranslation()
+  const {process} = props
+
+  const processAction = (lastAction: ProcessActionType): JSX.Element => {
+    const performedAt = formatRelatively(lastAction.performedAt)
+    const user = lastAction.user
+
+    return (
+      <span title={t("processLastAction.title", "Action run {{performedAt}} by {{user}}.", {performedAt, user})}>
```
Comment (mproch) [inline ui/client/components/Process/ProcessLastAction.tsx:21]: Performed by {{user}} at {{performedAt}}
?

File: ui/client/containers/Archive.tsx [hunk 3]
```diff
@@ -18,7 +18,8 @@ const ElementsRenderer: RowsRenderer = ({processes}) => processes.map(process =>
     <Td column="category">{process.processCategory}</Td>
     <Td column="createdBy" className="centered-column" value={process.createdBy}>{process.createdBy}</Td>
     <Td column="createdAt" className="centered-column" value={process.createdAt}><Date date={process.createdAt}/></Td>
-    <Td column="modifyDate" className="centered-column" value={process.modificationDate}><Date date={process.modificationDate}/></Td>
+    <Td column="actionDate" className="centered-column" value={process?.lastAction?.performedAt}><Date date={process?.lastAction?.performedAt}/></Td>
```
Comment (mproch) [inline ui/client/containers/Archive.tsx:21]: Is it obvious that for archived processes last action is Archive? Now it is so, but if we add other actions??

**Review by mproch — APPROVED** [review]
:+1: I added just one minor comment

File: ui/client/containers/Archive.tsx (hunk 3 above)
Comment (lciolecki) [reply ui/client/containers/Archive.tsx:21]: Right now we assume that after archive we can do only unarchvie :) If want change this behavior in future, then we should return this information in another fields.  Right now we can leave it.

**Pull request merged**